docs/EVE-LOGGING.md

```diff
@@ -35,3 +40,19 @@ When a direct ssh connection to a device is possible, /persist/rsyslog/syslog.tx
 ## Log export to cloud
 
 Rsyslogd is configured to send logs via TCP socket to logmanager. Logmanager bundles the logs into protobuf messages and exports them to cloud using API. Today when a device loses network connectivity to cloud, there is a back-off mechanism built into the logmanager that prompts rsyslogd to stop sending logs. When there is network drop, logmanager stops reading log messages from rsyslogd (TCP socket). Rsyslogd stops sending logs when it's output socket buffer is full. As a result following logs will be queued on disk. After the device's network connectivity is restored, rsyslogd starts sending logs from it's disk queues.
+
+## Log files still present in device
+
+1. Logmanager's log file present in /persist/log/logmanager.log
+   Latest code with cron, logrotate removed from pillar this log file grows irregulated.
+   What can be done to control the total size of this file on disk?
+
+   a. Bring back cron, logrotate?
+   b. Make logmanager send logs directly to /dev/log (not via memlogd) and put a programname
+      match in rsyslogd that redirects these logs into /persist/log/logmanger.log but using a
+	  size controlled out_channel.
+
+2. Reboot reason and reboot stack files present in /persist/IMGx and /persist/log directories.
```

Comment:
The ones in /persist/IMGx are overwritten, but the ones in /persist/log/ are concatenated, hence will grow.
But the concatenated ones we can get rid of once we've convinced ourselves that the logging support always manages to send the reason and stack to the EVE log API to the controller.

We should presumably also be able to move the files in /persist/IMGx to /persist/ since I don't think we need to track those per partition/IMGx. Maybe add the eve-release to the string we print to reboot-reason would be helpful to sanity check that it came from the software you thought you were running?

Reply:
Another tracker story for @gkodali-zededa coming right up! ;-)

Reply:
@eriknordmark @rvs Should I do the following for now?
1) Move reboot reason/stack from /persist/IMGx to /persist with the image partition tagged in the message itself
2) Keep /persist/log/reboot-reason/reboot-stack as they are for now.
Or did you mean something else?

Reply:
A. Keep the reboot*.log in /persist/log/ for now until we've validated that we never loose this infornation.
B. Move the /persist/IMGx/reboot* to /persist/ and tag the content with the eve-release (which is more important than the image partition). Note that zedagent grabs and clears that so it needs to be updated.

Reply:
DOne